include/na/Configuration.hpp

```diff
@@ -0,0 +1,46 @@
+//
+// This file is part of the MQT QMAP library released under the MIT license.
+// See README.md or go to https://github.com/cda-tum/mqt-qmap for more
+// information.
+//
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <istream>
+#include <string>
+#include <unordered_map>
+namespace na {
+
+enum class NaMappingMethod : std::uint8_t { NAIVE, SMART };
```

Comment:
Hm, very valid argument. I actually think `Naive` is fine, because it is *the* straightforward solution. I renamed the `Smart` one to `MaximizeParallelism` because I could not come up with something better for now. It general the smart method maximizes parallelism and minimizes the number of Load and Store operations. I find that hard to put into one name of a constant.